src/sidebar/components/ThreadCard.tsx

```diff
@@ -31,9 +31,7 @@ function ThreadCard({ frameSync, thread }: ThreadCardProps) {
   );
 
   const scrollToAnnotation = useCallback(
-    (ann: Annotation) => {
-      frameSync.scrollToAnnotation(ann);
-    },
+    (ann?: Annotation) => ann && frameSync.scrollToAnnotation(ann),
```

Comment:
This is called from two places now, so I have moved the check for `ann` to be defined here, to avoid duplicated conditional logic in two places.